templates/website.html

```diff
@@ -22,9 +22,16 @@
   </div>
   <div class="tabs">
     <ul class="nav nav-tabs">
+        {% if firstTabActive %}
```

Comment:
This condition could be inlined in the `li` as `<li{% if firstTabActive %} class="active"{% endif %}>` ;)